### ISSUE TITLE
App structure has changed

### ISSUE THREAD

**Issue by aendra-rininsland** [doronsever/generator-gulp-angular-subtask#11]
![Replace "framework" with "directory structure"](http://curtis.lassam.net/comics/cube_drone/76.gif)

`generator-gulp-angular` now uses the following directory structure:

```
src
├── app
│   ├── components
│   │   └── (directives and stuff)
│   └── main (controllers and stuff)
└── assets
    └── images
```

As such, `yo gulp-angular-subtask:directive directiveName` should populate src/app/components with a new directory (instead of components' parent directory). Further, using the navbar example from `generator-gulp-angular`, it should then be put in a flat folder structure (as opposed to creating `directiveName/directives/directiveName-directive.js` and `directiveName/spec/directives/directiveName-directive.spec.js`).

Will happily submit a PR if this change in behaviour is desirable.

**Comment by aendra-rininsland** [comment]
Update: Just noticed the `--component` flag. Any way it'd be possible to add that to the setup method to automatically put directives there if preferred?

**Comment by doronsever** [comment]
Hi
Sorry for the late response.

From what i understand, you would like a flag that say that all new modules will be put under the 'component' directory?

**Comment by aendra-rininsland** [comment]
@doronsever Yep, you got it. It's really annoying to forget that flag and then have to redo everything. Not sure how best to do that given this generator is only invoked after the main generator scaffolds everything, but it would be nice to have. Feel free to close this if you think that isn't possible.
